origin-request-edge-lambda/index.js

```diff
@@ -0,0 +1,68 @@
+var Netmask = require("netmask").Netmask;
+
+const redirects = require("./redirects.json").map(
+	({ source, destination }) => ({
+		source: new RegExp(source),
+		destination,
+	})
+);
+
+const allowedCounties = require("./allowedCounties.json").map(
+	(country) => country
+);
+
+const ipListJson = require("./ipAllowList.json");
+
+exports.handler = (event, context, callback) => {
+	let request = event.Records[0].cf.request;
+	const countryHeader = request.headers["cloudfront-viewer-country"];
+	const countryCode = countryHeader[0].value;
+
+	if (!isInCountryAllowList(countryCode)) {
+		//if request is from non UK country and not in ip allow list
+		const redirectResponse = {
+			status: "403",
+			statusDescription: "Forbidden",
+		};
+		callback(null, redirectResponse);
+	}
+
+	for (const { source, destination } of redirects) {
+		if (source.test(request.uri)) {
+			//if uri matches entry in rewrite list return modified uri
+			const redirectResponse = {
+				status: "302",
```

Comment:
the current redirects on CKS are all 301s e.g. https://github.com/nice-digital/cks-gatsby/blob/master/gatsby/static/.htaccess#L9

Reply:
remove status message